core/templates/dev/head/pages/exploration_editor/history_tab/CompareVersionsServiceSpec.js

```diff
@@ -13,156 +13,10 @@
-        cmd: 'AUTO_revert_version_number'
-      }],
-      version_number: 4
-    }, {
-      commit_type: 'edit',
-      commit_cmds: [{
-        cmd: 'delete_state',
-        state_name: 'B'
-      }, {
-        cmd: 'rename_state',
-        new_state_name: 'D',
-        old_state_name: 'A'
-      }],
-      version_number: 5
-    }, {
-      commit_type: 'revert',
-      commit_cmds: [{
-        version_number: 3,
-        cmd: 'AUTO_revert_version_number'
-      }],
-      version_number: 6
-    }, {
-      commit_type: 'edit',
-      commit_cmds: [{
-        cmd: 'add_state',
-        state_name: 'D'
-      }],
-      version_number: 7
-    }, {
-      commit_type: 'edit',
-      commit_cmds: [{
-        cmd: 'edit_state_property',
-        state_name: 'D',
-        new_value: {
-          html: 'Some text',
-          audio_translations: {}
-        },
-        old_value: {
-          html: '',
-          audio_translations: {}
-        }
-      }],
-      version_number: 8
-    }];
-
-    beforeEach(inject(function($injector) {
-      vts = $injector.get('versionsTreeService');
-    }));
-
-    it('should get correct list of parents', function() {
-      vts.init(snapshots);
-      var expectedParents = {
-        1: -1,
-        2: 1,
-        3: 2,
-        4: 2,
-        5: 4,
-        6: 3,
-        7: 6,
-        8: 7
-      };
-      expect(vts.getVersionTree()).toEqual(expectedParents);
-    });
-
-    it('should find correct LCA', function() {
-      vts.init(snapshots);
-      expect(vts.findLCA(1, 6)).toBe(1);
-      expect(vts.findLCA(3, 5)).toBe(2);
-      expect(vts.findLCA(3, 8)).toBe(3);
-      expect(vts.findLCA(3, 4)).toBe(2);
-      expect(vts.findLCA(3, 3)).toBe(3);
-      expect(vts.findLCA(2, 4)).toBe(2);
-    });
-
-    it('should get correct change list', function() {
-      vts.init(snapshots);
-      expect(function() {
-        vts.getChangeList(1);
-      }).toThrow(new Error('Tried to retrieve change list of version 1'));
-      expect(vts.getChangeList(2)).toEqual([{
-        cmd: 'add_state',
-        state_name: 'B'
-      }, {
-        cmd: 'rename_state',
-        new_state_name: 'A',
-        old_state_name: 'First State'
-      }]);
-      expect(vts.getChangeList(4)).toEqual([{
-        cmd: 'AUTO_revert_version_number',
-        version_number: 2
-      }]);
-      expect(vts.getChangeList(5)).toEqual([{
-        cmd: 'delete_state',
-        state_name: 'B'
-      }, {
-        cmd: 'rename_state',
-        new_state_name: 'D',
-        old_state_name: 'A'
-      }]);
-      expect(vts.getChangeList(8)).toEqual([{
-        cmd: 'edit_state_property',
-        state_name: 'D',
-        new_value: {
-          html: 'Some text',
-          audio_translations: {}
-        },
-        old_value: {
-          html: '',
-          audio_translations: {}
-        }
-      }]);
-    });
-  });
-});
-
-describe('Compare versions service', function() {
+describe('compare versions service', function() {
```

Comment:
Ah we don't have to change this, this describe block is different from the one inside it, so revert it back to `Compare versions service` only.